js/README.md

```diff
@@ -200,6 +200,13 @@ The JS community is a diverse group with a varied list of target environments an
 
 If you think we missed a compilation target and it's a blocker for adoption, please open an issue.
 
+### Supported Browsers and Platforms
+
+The bundles we compile support moderns browser released in the last 5 years. This includes supported versions of
+Firefox, Chrome, Edge, and Safari. We also support maintained versions of Node.
+We do not actively support Internet Explorer.
+Apache Arrow also works on [supported versions of Node](https://nodejs.org/en/about/releases/).
+
```

Comment:
nit: this mentions Node twice.
```suggestion
### Supported Browsers and Platforms

The bundles we compile support moderns browser released in the last 5 years. This includes supported versions of
Firefox, Chrome, Edge, and Safari. We do not actively support Internet Explorer.
Apache Arrow also works on [maintained versions of Node](https://nodejs.org/en/about/releases/).

```

Should we identify [Arrow.dom](http://arrow.apache.org/docs/js/modules/arrow_dom.html) for browsers vs [Arrow.node](http://arrow.apache.org/docs/js/modules/arrow_node.html) for node?

Reply:
Good catch. 

I think we should explain Arrow.dom and Arrow.node in the docs rather than putting too much into the readme.